Library/Homebrew/cmd/update-report.rb

```diff
@@ -206,6 +208,18 @@ def migrate_legacy_cache_if_necessary
     end
   end
 
+  def formula_resources(formula)
+    specs = [*formula.stable, *formula.devel, *formula.head]
```

Comment:
What's with all the `*`?

Reply:
Any of these could be `nil`, so `[*nil, *nil, *nil]` would be `[]`.

Reply:
Can you use `.compact` instead? That makes the intent a bit clearer.